src/OrchardCore.Modules/OrchardCore.Flows/Views/BagPart.cshtml

```diff
@@ -6,7 +6,7 @@
 <section class="flow">
     @foreach (var item in Model.BagPart.ContentItems)
     {
-        var itemContent = await ContentItemDisplayManager.BuildDisplayAsync(item, Model.BuildPartDisplayContext.Updater, Model.Settings.DisplayType ?? "Detail", Model.BuildPartDisplayContext.GroupId);
+        var itemContent = await ContentItemDisplayManager.BuildDisplayAsync(item, Model.BuildPartDisplayContext.Updater, Model.Settings.DisplayType ?? "Contained", Model.BuildPartDisplayContext.GroupId);
```

Comment:
I see what you are trying to do here @microposmp but I don't think you can introduce a new display type `Contained` here.

The drivers (part and field) only render content in a location of `Detail` or `Summary` (by default) so a new display type would likely render empty content here.

We suggested on the issue to simply add a couple of alternates at this point.

It would just need to be done in such a way that the alternates do not take precedence over the normal content item alternates (probably add to `Displaying`)

Reply:
Changed to add alternates in this file. I see the issue you describe with the drivers (part and field). I was not able to figure out how to see if the content type being rendered was contained in a BagPart.